crypto/pem/pem_local.h

```diff
@@ -113,11 +121,19 @@
                                             OUTTYPE, outtype, writename) \
     PEM_write_cb_fnsig(name, type, OUTTYPE, writename)                  \
     {                                                                   \
-        IMPLEMENT_PEM_provided_write_body_vars(type, asn1);             \
+        IMPLEMENT_PEM_provided_write_body_vars(type, asn1, NULL);       \
```

Comment:
Where is the libctx? It is in the macro def but not here.

Reply:
Ah, I apparently didn't fix up everything